sycl/test-e2e/Regression/pf-wg-atomic64.cpp

```diff
@@ -0,0 +1,27 @@
+// REQUIRES: accelerator
```

Comment:
This test implies that accelerator doesn't support `atomic64` aspect. While this is true for our current SW/HW available, it technically may change in the future.

I would maybe suggest to add a runtime check into the test as well, so it skips itself if `atomic64` is supported to avoid seeing sudden unrelated/unexpected failures in the future.

If I understand correctly, we can even do something like: `UNSUPPORTED: aspect-atomic64` to get it automatically skipped on devices which support `atomic64`. Perhaps it is better to use that line instead of `REQUIRES`, because it will enable a better coverage